{"uid":"a31b8f900bc4e7c3","name":"testCreateFirstBudgets","fullName":"test.BudgetsTest.testCreateFirstBudgets","historyId":"5a843a6ac14f21b975241eceb40bb25a","time":{"start":1721689680683,"stop":1721689680684,"duration":1},"description":"Create Budgets FI-T5","descriptionHtml":"<p>Create Budgets FI-T5</p>\n","status":"skipped","statusMessage":"java.lang.RuntimeException: Something went wrong while clearing data 404","statusTrace":"java.lang.RuntimeException: java.lang.RuntimeException: Something went wrong while clearing data 404\n\tat utils.run.ClearData.deleteHttp(ClearData.java:81)\n\tat utils.run.ClearData.deleteAccounts(ClearData.java:109)\n\tat utils.run.ClearData.clearData(ClearData.java:128)\n\tat utils.run.BaseTest.clearData(BaseTest.java:54)\n\tat utils.run.BaseTest.beforeMethod(BaseTest.java:74)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\nCaused by: java.lang.RuntimeException: Something went wrong while clearing data 404\n\tat utils.run.ClearData.deleteHttp(ClearData.java:77)\n\t... 41 more\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"beforeMethod","time":{"start":1721689680195,"stop":1721689680586,"duration":391},"description":"","status":"broken","statusMessage":"java.lang.RuntimeException: Something went wrong while clearing data 404","statusTrace":"java.lang.RuntimeException: java.lang.RuntimeException: Something went wrong while clearing data 404\n\tat utils.run.ClearData.deleteHttp(ClearData.java:81)\n\tat utils.run.ClearData.deleteAccounts(ClearData.java:109)\n\tat utils.run.ClearData.clearData(ClearData.java:128)\n\tat utils.run.BaseTest.clearData(BaseTest.java:54)\n\tat utils.run.BaseTest.beforeMethod(BaseTest.java:74)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\nCaused by: java.lang.RuntimeException: Something went wrong while clearing data 404\n\tat utils.run.ClearData.deleteHttp(ClearData.java:77)\n\t... 41 more\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[{"name":"afterMethod","time":{"start":1721689680686,"stop":1721689680686,"duration":0},"description":"","status":"broken","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"package","value":"test.BudgetsTest"},{"name":"testClass","value":"test.BudgetsTest"},{"name":"testMethod","value":"testCreateFirstBudgets"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"test.BudgetsTest"},{"name":"host","value":"fv-az1144-364"},{"name":"thread","value":"2941@fv-az1144-364.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Budgets"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":18,"passed":6,"unknown":0,"total":24},"items":[{"uid":"b1180797408735cd","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/113//#testresult/b1180797408735cd","status":"skipped","statusDetails":"java.net.UnknownHostException: app: Temporary failure in name resolution","time":{"start":1721670613277,"stop":1721670613278,"duration":1}},{"uid":"8d65dc03b57bfcb2","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/112//#testresult/8d65dc03b57bfcb2","status":"skipped","statusDetails":"java.lang.RuntimeException: Something went wrong while clearing data io.restassured.internal.ValidatableResponseImpl@2da273b3","time":{"start":1721669970097,"stop":1721669970098,"duration":1}},{"uid":"1b312f0028da5be7","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/111//#testresult/1b312f0028da5be7","status":"skipped","statusDetails":"java.lang.RuntimeException: Something went wrong while clearing dataio.restassured.internal.ValidatableResponseImpl@1c8d9ac9","time":{"start":1721669032946,"stop":1721669032947,"duration":1}},{"uid":"6c9f79bb7d10cd38","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/110//#testresult/6c9f79bb7d10cd38","status":"skipped","statusDetails":"java.lang.RuntimeException: Something went wrong while clearing dataio.restassured.internal.ValidatableResponseImpl@5cb42eb2","time":{"start":1721668614763,"stop":1721668614764,"duration":1}},{"uid":"b3e4c3b059966cfe","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/109//#testresult/b3e4c3b059966cfe","status":"skipped","statusDetails":"java.lang.RuntimeException: Something went wrong while clearing dataio.restassured.internal.ValidatableResponseImpl@387bbd7a","time":{"start":1721667385056,"stop":1721667385057,"duration":1}},{"uid":"283acf1e2b04a3c2","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/109//#testresult/283acf1e2b04a3c2","status":"skipped","statusDetails":"java.lang.RuntimeException: Something went wrong while clearing dataio.restassured.internal.ValidatableResponseImpl@3003d288","time":{"start":1721666693673,"stop":1721666693673,"duration":0}},{"uid":"b40486b2a2dd8a03","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/108//#testresult/b40486b2a2dd8a03","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:","time":{"start":1721601636892,"stop":1721601636893,"duration":1}},{"uid":"15e798c4504a2928","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/107//#testresult/15e798c4504a2928","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:","time":{"start":1721600923524,"stop":1721600923525,"duration":1}},{"uid":"e9d2e4bae500fe34","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/106//#testresult/e9d2e4bae500fe34","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:","time":{"start":1721599102721,"stop":1721599102722,"duration":1}},{"uid":"ca7a2db33dfc400a","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/105//#testresult/ca7a2db33dfc400a","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:","time":{"start":1721598071460,"stop":1721598071461,"duration":1}},{"uid":"4f68b4f9fb29f459","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/104//#testresult/4f68b4f9fb29f459","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:","time":{"start":1721596041347,"stop":1721596041348,"duration":1}},{"uid":"bed5f455580c44a8","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/103//#testresult/bed5f455580c44a8","status":"skipped","statusDetails":"java.lang.RuntimeException: Authorization does not work with token:null","time":{"start":1721594892453,"stop":1721594892454,"duration":1}},{"uid":"4c67d90141cb416a","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/102//#testresult/4c67d90141cb416a","status":"passed","time":{"start":1721421880886,"stop":1721421881839,"duration":953}},{"uid":"52ac66e39c296a44","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/101//#testresult/52ac66e39c296a44","status":"skipped","statusDetails":"java.lang.NullPointerException: Cannot invoke \"com.fasterxml.jackson.databind.JsonNode.iterator()\" because \"dataNode\" is null","time":{"start":1721420875987,"stop":1721420875988,"duration":1}},{"uid":"597005657f9dff5e","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/100//#testresult/597005657f9dff5e","status":"skipped","statusDetails":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[@type='token']\"}\n  (Session info: chrome=126.0.6478.126)\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nBuild info: version: '4.22.0', revision: 'c5f3146703*'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1023-azure', java.version: '17.0.12'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [aae378c80a675714c507d171dcf48031, findElement {value=//button[@type='token'], using=xpath}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.126, chrome: {chromedriverVersion: 126.0.6478.182 (5b5d8292ddf..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:41767}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:41767/devtoo..., se:cdpVersion: 126.0.6478.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nSession ID: aae378c80a675714c507d171dcf48031","time":{"start":1721420222462,"stop":1721420222463,"duration":1}},{"uid":"94dd283bd4363567","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/99//#testresult/94dd283bd4363567","status":"skipped","statusDetails":"no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//button[@type='test@gmail.com']\"}\n  (Session info: chrome=126.0.6478.126)\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nBuild info: version: '4.22.0', revision: 'c5f3146703*'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1023-azure', java.version: '17.0.12'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [06198bd36ebf00142ea395724b8ae7a2, findElement {using=xpath, value=//button[@type='test@gmail.com']}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.126, chrome: {chromedriverVersion: 126.0.6478.182 (5b5d8292ddf..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:44273}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:44273/devtoo..., se:cdpVersion: 126.0.6478.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nSession ID: 06198bd36ebf00142ea395724b8ae7a2","time":{"start":1721418200808,"stop":1721418200809,"duration":1}},{"uid":"8f91388f7b179f06","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/98//#testresult/8f91388f7b179f06","status":"skipped","statusDetails":"unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: chrome=126.0.6478.126)\nBuild info: version: '4.22.0', revision: 'c5f3146703*'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1023-azure', java.version: '17.0.12'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [9392fd5e58d95fc84dae523c60b34cbb, get {url=http://null/}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.126, chrome: {chromedriverVersion: 126.0.6478.182 (5b5d8292ddf..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:33165}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:33165/devtoo..., se:cdpVersion: 126.0.6478.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nSession ID: 9392fd5e58d95fc84dae523c60b34cbb","time":{"start":1721417368720,"stop":1721417368721,"duration":1}},{"uid":"b0c1a0b154279866","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/97//#testresult/b0c1a0b154279866","status":"skipped","statusDetails":"unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: chrome=126.0.6478.126)\nBuild info: version: '4.18.1', revision: 'b1d3319b48'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1023-azure', java.version: '17.0.12'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [e12afbe71908aa377445fa6cdd65b200, get {url=http://null/}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.126, chrome: {chromedriverVersion: 126.0.6478.182 (5b5d8292ddf..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:38613}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:38613/devtoo..., se:cdpVersion: 126.0.6478.126, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nSession ID: e12afbe71908aa377445fa6cdd65b200","time":{"start":1721416805518,"stop":1721416805519,"duration":1}},{"uid":"7dfcff2729bc1126","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/96//#testresult/7dfcff2729bc1126","status":"passed","time":{"start":1711222240071,"stop":1711222240960,"duration":889}},{"uid":"b1f265d67690a45f","reportUrl":"https://Vlad1kek.github.io/Firefly_III_Test_Automation/95//#testresult/b1f265d67690a45f","status":"passed","time":{"start":1711221902487,"stop":1711221903382,"duration":895}}]},"tags":[]},"source":"a31b8f900bc4e7c3.json","parameterValues":[]}